### ISSUE TITLE
[A11Y] "Review these error" link is not reachable

### ISSUE THREAD

**Issue by Larisa-Staroverova** [SAP/spartacus#11990]

**Comment by Larisa-Staroverova** [comment]
Known issue regarding [event.preventDefault()](https://stackoverflow.com/questions/33044607/if-i-set-a-break-point-in-keydown-event-the-key-i-typed-wont-be-appended-to-th)

**Comment by Changsuwan** [comment]
We exclude the navigation with space key on the link due to this conversation: https://sap-cx.slack.com/archives/CMBTFUYVA/p1618932777130000
